app/controllers/draft_events_controller.rb

```diff
@@ -0,0 +1,34 @@
+class DraftEventsController < WebsocketRails::BaseController
+  def initialize_session
+    # perform application setup here
+    controller_store[:active_player_index] = 0
+    controller_store[:round_rotation] = [
+        [3, Draft::ROUND_TYPE[0]],
```

Comment:
Use 2 spaces for indentation in an array, relative to the start of the line where the left bracket is.